src/subscriptions/walletConnect.ts

```diff
@@ -0,0 +1,177 @@
+import { WalletConnectProvider, BufferBlock, Block } from '../provider/types'
```

Comment:
How do you generated this typescript?
For me is relevant, among other things to know how we update it in future versions of WalletConnect for example

Reply:
WalletConnect doesn't provider many types as of yet, not to mention it's in beta. So we will have to change if they change.
That is true for any change in any underlying library, so nothing new.

Block and Buffer block are modeled after what is returned from blockchain as a payload so it's more or less fixed.